src/Datadog.Trace/IScopeManager.cs

```diff
@@ -2,22 +2,53 @@
 
 namespace Datadog.Trace
 {
-    internal interface IScopeManager
+    /// <summary>
+    /// Interface for managing a scope.
+    /// </summary>
+    public interface IScopeManager
```

Comment:
Any particular reason we changed this to `public`?

Reply:
Can't remember exactly -- a number of changes were required when I looked at it last week, maybe I went too far. Rebuilt and it builds with internal so I'm changing it back.

Reply:
reverted back to internal